shared-helpers/src/locales/tl.json

```diff
@@ -74,6 +74,7 @@
   "application.contact.apt": "Apt o Unit #",
   "application.contact.cityName": "Pangalan ng Lungsod",
   "application.contact.city": "Lungsod",
+  "application.contact.county": "Lalawigan / County",
```

Comment:
are these placeholders? not sure how i18n is going to be handled (either through us just using google translate, or we get some translators in)

Reply:
Exactly. These are placeholders and when we finally have the translation services worked out, we'll add these on that list to be verified.